src/python_testing/TC_BOOLCFG_4_3.py

```diff
@@ -0,0 +1,330 @@
+
+        self.step("2b")
+        attribute_list = await self.read_boolcfg_attribute_expect_success(endpoint=endpoint, attribute=attributes.AttributeList)
+
+        self.step(3)
+        if attributes.AlarmsEnabled.attribute_id not in attribute_list:
+            logging.info("AlarmsEnabled not supported skipping test case")
+
+            # Skipping all remainig steps
+            self.step(4)
+            logging.info("Test step skipped")
+            self.step("5a")
+            logging.info("Test step skipped")
+            self.step("5b")
+            logging.info("Test step skipped")
+            self.step(6)
+            logging.info("Test step skipped")
+            self.step(7)
+            logging.info("Test step skipped")
+            self.step(8)
+            logging.info("Test step skipped")
+            self.step(9)
+            logging.info("Test step skipped")
+            self.step(10)
+            logging.info("Test step skipped")
+            self.step(11)
+            logging.info("Test step skipped")
+            self.step(12)
+            logging.info("Test step skipped")
+            self.step(13)
+            logging.info("Test step skipped")
+            self.step(14)
+            logging.info("Test step skipped")
+            self.step("15a")
+            logging.info("Test step skipped")
+            self.step("15b")
+            logging.info("Test step skipped")
+            self.step(16)
+            logging.info("Test step skipped")
+            self.step(17)
+            logging.info("Test step skipped")
+            self.step(18)
+            logging.info("Test step skipped")
+            self.step(19)
+            logging.info("Test step skipped")
+            self.step(20)
+            logging.info("Test step skipped")
+            self.step(21)
+            logging.info("Test step skipped")
+            self.step(22)
+            logging.info("Test step skipped")
+            self.step(23)
+            logging.info("Test step skipped")
+            self.step(24)
+            logging.info("Test step skipped")
+
+            return
+        else:
+            logging.info("Test step skipped")
+
+        self.step(4)
+        enabledAlarms = 0
+
+        self.step("5a")
+        if is_vis_feature_supported:
+            enabledAlarms |= Clusters.BooleanStateConfiguration.Bitmaps.AlarmModeBitmap.kVisual
+        else:
+            logging.info("Test step skipped")
+
+        self.step("5b")
+        try:
+            await self.send_single_cmd(cmd=Clusters.Objects.BooleanStateConfiguration.Commands.EnableDisableAlarm(alarmsToEnableDisable=enabledAlarms), endpoint=endpoint)
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, Status.Success, "Unexpected error returned")
+            pass
+
+        test = await self.read_boolcfg_attribute_expect_success(endpoint=endpoint, attribute=attributes.AlarmsEnabled)
+        print(enabledAlarms)
```

Comment:
probably want to remove these - I think this might be leftover debug

Reply:
Oh yes. Thanks for catching. I though I had removed all of these 😅